swift_kernel.py

```diff
@@ -448,6 +539,16 @@ def _install_packages(self, packages, swiftpm_flags):
                     'Install Error: Cannot install packages because '
                     'SWIFT_IMPORT_SEARCH_PATH is not specified.')
 
+        install_lock_file = os.path.join(swift_import_search_path, 'install_lock')
```

Comment:
I was thinking of a lock that locks things until you shutdown the kernel, so that a second notebook doesn't change the installed packages from under you while you're running the notebook.

But that will be pretty complicated to implement:
* How to make sure that the lock file gets removed even if the kernel shuts down uncleanly?
* Perhaps we should allow multiple notebooks to use the same install dir, as long as they use the same packages. This will require the locking to understand what packages are installed.

I think locking is not essential, so I suggest removing the current locking code and deferring locking for later (or never).

Reply:
> I think locking is not essential, so I suggest removing the current locking code and deferring locking for later (or never).

Agreed.